src/WebJobs.Extensions.DurableTask/EntityScheduler/EntityId.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
     /// </summary>
     public struct EntityId : IEquatable<EntityId>, IComparable
     {
+        private string schedulerId;
```

Comment:
Should this be `readonly`?

Reply:
Can no longer be readonly now that I realized that the constructor is not the only path in which this field needs to be populated.